lib/ast-utils.js

```diff
@@ -0,0 +1,176 @@
+/**
+ * @fileoverview Common utils for AST.
+ * @author Gyandeep Singh
+ * @copyright 2015 Gyandeep Singh. All rights reserved.
+ */
+
+"use strict";
+
+/**
+ * Finds a JSDoc comment node in an array of comment nodes.
+ * @param {ASTNode[]} comments The array of comment nodes to search.
+ * @param {int} line Line number to look around
+ * @returns {ASTNode} The node if found, null if not.
+ * @private
+ */
+function findJSDocComment(comments, line) {
+
+    if (comments) {
+        for (var i = comments.length - 1; i >= 0; i--) {
+            if (comments[i].type === "Block" && comments[i].value.charAt(0) === "*") {
+
+                if (line - comments[i].loc.end.line <= 1) {
+                    return comments[i];
+                } else {
+                    break;
+                }
+            }
+        }
+    }
+
+    return null;
+}
+
+/**
+ * Check to see if its a ES6 export declaration
+ * @param {ASTNode} astNode - any node
+ * @returns {boolean} whether the given node represents a export declaration
+ * @private
+ */
+function looksLikeExport(astNode) {
+    return astNode.type === "ExportDefaultDeclaration" || astNode.type === "ExportNamedDeclaration" ||
+        astNode.type === "ExportAllDeclaration" || astNode.type === "ExportSpecifier";
+}
+
+/**
+ * Gets all comments for the given node.
+ * @param {ASTNode} node The AST node to get the comments for.
+ * @returns {Object} The list of comments indexed by their position.
+ * @public
+ */
+module.exports.getComments = function(node) {
```

Comment:
Almost there. :) As I said initially, the project uses this format:

```js
module.exports = {

}
```

Sorry for nitpicking, but consistency within the project is important.